src/main/java/db/MongoDBManager.java

```diff
@@ -52,8 +56,8 @@ public void close(){
     public ArrayList<String> likeFilterByCriteria(String collection, String field,String criteria) {
         final String fieldName = field;
         final ArrayList<String> list = new ArrayList<String>();
-        MongoCollection<Document> roomsCollation = MongoDBManager.getInstance().getCollection(collection);
-        FindIterable<Document> rooms = roomsCollation
+        MongoCollection<Document> documentsCollation = MongoDBManager.getInstance().getCollection(collection);
```

Comment:
MongoCollection<Document> documentsCollation = getCollection(collection); i think that you do not need to use MongoDBManager.getInstance() because you are in the MongoDBManager class